client/src/__tests__/App.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { act, render } from '@testing-library/react';
+import 'mobx-react-lite/batchingForReactDom';
+import React from 'react';
+import api from '~/api/test';
+import { App } from '~/components/App';
+import { StoreProvider } from '~/store';
+
+test('App renders without exceptions', async () => {
```

Comment:
This test actually does nothing, since code generated from jsx can hardly have any errors lead. If it does, the world is collapsing :)

Reply:
What do you mean? This test actually renders the whole app with mocked data.

Reply:
Discussed offline, decided to leave it here